TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -153,15 +153,15 @@ class MetAPI{
             parameters: [ : ]) { result in
                 switch result {
                 case .failure:
-                    completion(.failure(.invalidUrlComponents))
+                    completion(.failure(.networkError(.invalidUrlComponents)))
```

Comment:
This is not a `invalidUrlComponents` of the `networkError`, please use the provided value